public/js/icinga/behavior/navigation.js

```diff
@@ -354,6 +360,40 @@
         $(window).trigger('resize');
     };
 
+    /**
+     * Toggle config flyout visibility
+     *
+     * @param {Object} e Event
+     */
+    Navigation.prototype.toggleConfigFlyout = function(e) {
+        var _this = e.data.self;
+        if ($('#layout').is('.config-flyout-open')) {
+            _this.hideConfigFlyout(e);
+        } else {
+            _this.showConfigFlyout(e);
+        }
+    }
+
+    /**
+     * Hide config flyout
+     *
+     * @param {Object} e Event
+     */
+    Navigation.prototype.hideConfigFlyout = function(e) {
+        $('#layout').removeClass('config-flyout-open');
+        delete $(e.target).closest('.container')[0].dataset.suspendAutorefresh;
+    }
+
+    /**
+     * Show config flyout
+     *
+     * @param {Object} e Event
+     */
+    Navigation.prototype.showConfigFlyout = function(e) {
+        $('#layout').addClass('config-flyout-open');
+        $(e.target).closest('.container')[0].dataset.suspendAutorefresh = 1;
```

Comment:
Not `1`, the empty string, that was not a typo! This way it results in a true boolean data attribute.

Reply:
✅